tests/integration/test_fim/test_files/test_file_limit/test_file_limit_capacity_alerts.py

```diff
@@ -118,8 +119,8 @@ def get_configuration(request):
 
 
 @pytest.mark.parametrize('percentage', [(0), (80), (90)])
-def test_file_limit_capacity_alert(percentage, get_configuration, configure_environment, restart_syscheckd,
-                                   wait_for_fim_start):
+def test_file_limit_capacity_alert(percentage, configure_local_internal_options_module, get_configuration,
+                                   configure_environment, restart_syscheckd, wait_for_fim_start):
```

Comment:
You need to add the `configure_local_internal_options_module` inside parameters documentation section.

Reply:
Done in commit # https://github.com/wazuh/wazuh-qa/pull/3280/commits/4a7f9c8fcbc40be0390b95e84d41d6c802cd981c